en/docs/learn/admin-api.md

```diff
@@ -0,0 +1,273 @@
+Sign in to the API Publisher as a user whose role includes `Internal/publisher`. Follow the steps given below: 
+
+1. Sign in to the API Publisher portal at `https:// <WSO2_OB_APIM_HOST>:9443/publisher` with `creator/publisher` privileges.
+   You can use the credentials for `mark@gold.com`.
+     ![sign_into](../assets/img/get-started/quick-start-guide/sign-in.png)
+
+2. In the homepage, go to **REST API** and select **Import Open API**. 
+     ![let's_get_started](../assets/img/get-started/quick-start-guide/lets-get-started.png)
+
+3. Select **OpenAPI File/Archive**. ![create-an-api](../assets/img/get-started/quick-start-guide/create-an-api.png)
+
+4. Click **Browse File to Upload** and select the `<APIM_HOME>/<OB_APIM_TOOLKIT_HOME>/repository/resources/apis/consumerdatastandards.org.au/CDSAdminAPIs/1.8.0/cds-admin-1.8.0.yaml` file.
+
+5. Click **Next**.
+
+6. Set the **Endpoint** as follows:
+     ```
+     https://<WSO2_OB_APIM_HOST>:9443/cds-admin-api/au100
+     ```
+7. Click **Create** to create the API
+
+8. After the API is successfully created, go to **Portal Configurations** using the left menu panel.
+     ![portal-configurations](../assets/img/get-started/quick-start-guide/portal-configurations.png)
+
+9. Select **Subscriptions** from the left menu pane and set the business plan to **Unlimited: Allows unlimited requests**. 
+     ![business-plan](../assets/img/get-started/quick-start-guide/business-plan.png)
+
+10. Click **Save**.
+
+11. Once you get the message that the API is successfully updated, use the left menu panel and select **API Configurations > Runtime**.
+      
+     ![select_runtime](../assets/img/get-started/quick-start-guide/select-runtime.png)
+
+12. Click the **Edit** button under **Request > Message Mediation**. ![edit_message_mediation](../assets/img/get-started/quick-start-guide/edit-message-mediation.png)
+
+13. Now, select the **Custom Policy** option.
+
+14. Upload the `<APIM_HOME>/<OB_APIM_TOOLKIT_HOME>/repository/resources/apis/consumerdatastandards.org.au/CDSAdminAPIs/1.8.0/cds-admin-endpoint-insequence-1.8.0.xml` file.
+
+15. Click **Select**.
+
+16. Scroll down and click **SAVE**.
+
+17. Go to **Deployments** using the left menu pane.
+     
+      ![select_deployments](../assets/img/get-started/quick-start-guide/select-deployments.png)
+
+18. Select the API Gateway type 
+
+      ![api_gateway](../assets/img/get-started/quick-start-guide/dcr-api-gateway.png)
+  
+19. Click **Deploy**.
+
+20. Go to **Overview** using the left menu pane.
+      
+      ![select_overview](../assets/img/get-started/quick-start-guide/select-overview.png)
+
+21. Click **Publish**. 
+
+      ![publish_api](../assets/img/get-started/quick-start-guide/publish-api.png)
+
+## Invoke Metrics API
+
+### GET Metrics - GET /admin/metrics
+
+This end point allows the Australian Competition & Consumer Commission to obtain operational statistics from the 
```

Comment:
endpoint is a single word. 
https://docs.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/e/endpoint